ui/StatusQ/sanity_checker/CMakeLists.txt

```diff
@@ -1,34 +1,13 @@
-project(SanityChecker)
+project(sanity_checker)
```

Comment:
Any specific reason why this is renamed?

Reply:
Good question!
I certainly remember that I saw CMake naminv conventions pointing that `snake_case` should be used for target names. But I couldn't find it now 😐

Reply:
Let me just drop this change, it's definitely out of scope on this PR anyway.